src/math/stats.cr

```diff
@@ -0,0 +1,11 @@
+module Math
+  extend self
+  
+  def mean(value : StaticArray | Array | Tuple)
+    value.sum / value.length
+  end
+
+  def mean(value : Enumerable)
```

Comment:
@zamith thanks for looking at the code. I haven't found yet the time to work on this, would you be interested in doing some work on it? It may be better to close this PR anyhow, as I think the maths/stats routines are intended to go in a library separate from the main crystal distribution.